### ISSUE TITLE
Archive can not be opened again after inserting a figure

### ISSUE THREAD

**Issue by oliver----** [substance/texture#884]
Reported by @JGilbert-eLife, need to investigate.

**Comment by JGilbert-eLife** [comment]
More detail - if I add in figures, I cannot reopen my work after saving it and exiting. I get the following error when I try to open my document:

ERROR:ParserError: <parsererror xmlns="http://www.w3.org/1999/xhtml" style="display: block; white-space: pre; border: 2px solid #c77; padding: 0 1em 0 1em; margin: 1em; background-color: #fdd; color: black"><h3>This page contains the following errors:</h3><div style="font-family:monospace;font-size:12px">error on line 2 at column 6: XML declaration allowed only at the start of the document </div><h3>Below is a rendering of the page up to the first error.</h3></parsererror>

**Comment by oliver----** [comment]
Ok thanks!

I need to take a closer look. Seems that we are writing out not even XML.

**Comment by JGilbert-eLife** [comment]
This ended up in the XML:
```

<fig id="_figure-1">
          <label>Figure 1</label>
          <caption id="caption-2646c4a4fe6f964cda52fb5f8f97ae39">
            <title>Figure test</title>
            <p id="p-34c78a015bd3c285969ceb7bd351bb96">Testing </p>
          </caption>
          <graphic id="_graphic-1" mime-subtype="png" mimetype="image" xlink:href="25d841d1be04366121330a126e1b328b.png" />
        </fig>
```
however, the manifest file is now showing two XML declarations!
```

<?xml version="1.0" encoding="UTF-8"?>
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE manifest PUBLIC "RDC-Manifest 1.0" "//Manifest-1.0.dtd">
<dar id="dar-1">
  <documents id="documents-1">
    <document id="manuscript" type="article" path="manuscript.xml" />
  </documents>
  <assets id="assets-1">
    <asset id="25d841d1be04366121330a126e1b328b" path="25d841d1be04366121330a126e1b328b.png" type="image/png" />
  </assets>
</dar>
```

**Comment by oliver----** [comment]
Ok... the manifest is broken.

**Comment by JGilbert-eLife** [comment]
Deleting the second declaration means I can open it fine.

**Comment by oliver----** [comment]
Sweet :)
Thanks!

**Comment by oliver----** [comment]
Know how to fix this.

**Comment by oliver----** [comment]
Working on this in #897

**Comment by oliver----** [comment]
Fixed via #897
